umh-core/pkg/fsm/s6/actions.go

```diff
@@ -129,26 +129,8 @@ func (s *S6Instance) initiateS6Stop(ctx context.Context, filesystemService files
 	return nil
 }
 
-// InitiateS6Restart attempts to restart the S6 service.
```

Comment:
Was this restart action unused ?
Because i cant find any replacement for it in your new code.

Reply:
yes it was unused